### PR TITLE
Remove Is Scheduled

### PR DESCRIPTION
**Pull request by michaelwehrley**
Remove Is Scheduled

* Add helper compute percents and display message dealing with  
  the number of reviwed & completed activities relative the number  
  of activities that were at one time planned.
* Add helper specs.
* Add scope ```were_planned``` to query all activities that were  
  at one time planned.
* Update providers by removing ```is_scheduled``` from parameters.
* Add migration to drop ```is_scheduled``` from activities table.
* Update activity fixtures to reflect removal of ```is_scheduled```.

[#89418002, #89044666]

### CODE REVIEW

File: app/helpers/think_feel_do_engine/activities_helper.rb [hunk 1]
```diff
@@ -15,5 +15,28 @@ def average_intensity_difference(activities, attribute)
...
+      "(You completed "\
+      "#{activities.reviewed_and_complete.count} "\
+      "out of #{scheduled_message(activities.were_planned)} "\
+      "that you scheduled.)"
+    end
+
+    def percent_complete(activities)
+      (
+        activities.reviewed_and_complete.count.to_f /
+        (activities.were_planned.count).to_f * 100
```
Comment (ericcf) [inline app/helpers/think_feel_do_engine/activities_helper.rb]: @michaelwehrley you'll have a problem here if `activities.were_planned.count == 0`!

**Comment by eschlange** [comment]
:+1:

File: app/helpers/think_feel_do_engine/activities_helper.rb (hunk 1 above)
Comment (michaelwehrley) [inline app/helpers/think_feel_do_engine/activities_helper.rb]: Ahh! Arg!

**Comment by michaelwehrley** [comment]
@ericcf Not sure if because I rebased and forced this branch if my latest update to this pull request is going to run on travis.  Thanks!

**Pull request merged**